src/main/java/com/vinylteam/vinyl/web/servlets/SignInServlet.java

```diff
@@ -55,7 +61,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                 response.setStatus(HttpServletResponse.SC_OK);
                 log.debug("Set response status to {'status':{}}", HttpServletResponse.SC_OK);
                 User user = optionalUser.get();
-                HttpSession session = request.getSession(true);
+                session = request.getSession(true);
                 session.setMaxInactiveInterval(60 * 60 * 5);
```

Comment:
Please remove magic number. Add it to application.properties and this class fields.